unit/unit.proto

```diff
@@ -24,20 +24,21 @@ message Key {
 	// CommitID is the commit ID of the repository containing this
 	// source unit, if any. The scanner tool need not fill this in; it
 	// should be left blank, to be filled in by the `srclib` tool.
-	//
-	// CommitID can also be a semantic version if the unit is unresolved.
 	string CommitID = 2;
 
+	// Version is the unresolved source unit version (e.g., "v1.2.3").
```

Comment:
can you expand on when this will be empty/etc like you do with the other fields?